lib/src/features/music_recognition/repositories/local_recognition_repository.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:sequence/src/datasource/local/converters/apple_music_db.dart';
+import 'package:sequence/src/datasource/local/converters/spotify_db.dart';
+import 'package:sequence/src/datasource/local/database.dart';
+import 'package:sequence/src/datasource/models/responses/recognition_response/recognition_result.dart';
+import 'package:sequence/src/datasource/models/responses/recognition_response/spotify_result/album.dart';
+
+class LocalRecognitionRepository {
+  final db = AppDatabase();
+
+  Future<String> saveRecognitions(RecognitionResult response) async {
+    try {
+      await db.into(db.recognitionResultDb).insert(
+            RecognitionResultDbCompanion.insert(
+              artist: response.artist,
+              title: response.title,
+              releaseDate: response.releaseDate,
+              label: response.label,
+              timeCode: response.timecode,
+              songLink: response.songLink,
+              album: response.album ?? '',
+              appleMusic: AppleMusicDb(
```

Comment:
**suggestion (non blocking)**: Create an extension on the Apple Music Result to map this

Reply:
Thank you for the suggestion, right now I do not know how to do that or fully understand how you'd like me to, would you kindly show me how to for future cases?